### ISSUE TITLE
CRUD Tutorial 4.0 incorrect import model

### ISSUE THREAD

**Issue by adriallongarriu** [Laravel-Backpack/Docs#72]
I was getting an error following the tutorial CRUD Tutorial 4.0 in a new project [Model](https://backpackforlaravel.com/docs/4.0/crud-tutorial) 

Import for CrudTrait is incorrect  `Backpack\CRUD\CrudTrait;` 

Solution from [Upgrade Guide](https://backpackforlaravel.com/docs/4.0/upgrade-guide)

> replace `Backpack\CRUD\CrudTrait` with `Backpack\CRUD\app\Models\Traits\CrudTrait`

**Comment by tabacitu** [comment]
Indeed! Thank you for spotting this, and letting us know about it @adriallongarriu ! It's now fixed.

Cheers!
